src/assessments/seo/SubHeadingsKeywordAssessment.js

```diff
@@ -62,7 +82,12 @@ export default class SubHeadingsKeywordAssessment extends Assessment {
 	 * @returns {boolean} True when there is at least one subheading.
 	 */
 	hasSubheadings( paper ) {
-		const subheadings = getSubheadings( paper.getText() );
+		let subheadings;
+		if ( process.env.YOAST_RECALIBRATION === "enabled" ) {
+			subheadings = getSubheadingsTopLevel( paper.getText() );
+		} else {
+			subheadings = getSubheadings( paper.getText() );
+		}
```

Comment:
Originally I left this as an explicit if statement because it is the beta and the tree shaking should remove the block that is never used. However, we do not apply this like that in a lot of other cases. Also, it is very possible that the tree shaking can shake a ternary statement perfectly.

**TL;DR** Will change! 😄